runtime/turing/src/lib.rs

```diff
@@ -36,7 +36,7 @@ use sp_runtime::{
 	transaction_validity::{TransactionSource, TransactionValidity},
 	AccountId32, ApplyExtrinsicResult, DispatchError, ModuleError, Percent, RuntimeDebug,
 };
-use xcm::{
+pub use xcm::{
```

Comment:
Why is this `pub`?

Reply:
Since xcm::v1::Junction::Parachain and xcm::latest::prelude::X1 will be used in node/src/chain_spec/turing.rs, they need to be exposed.

This is similar to mangata's code.

https://github.com/mangata-finance/mangata-node/blob/25b2af6d95de3399033a8ebaf05a6a9dfd78c1cb/runtime/mangata-rococo/src/lib.rs#L66

https://github.com/mangata-finance/mangata-node/blob/25b2af6d95de3399033a8ebaf05a6a9dfd78c1cb/node/src/chain_spec/mangata_rococo.rs#L7

Reply:
This is unnecessary indirection/abstraction. These deps can be pulled directly from the originating xcm crate rather than the runtime file- https://github.com/OAK-Foundation/OAK-blockchain/blob/69eb46e21c06504497c340a0a391d00b7a53c1e3/node/src/chain_spec/turing.rs#L17-L20

Reply:
This is unnecessary indirection/abstraction. These deps can be pulled directly from the originating xcm crate rather than the runtime file- https://github.com/OAK-Foundation/OAK-blockchain/blob/69eb46e21c06504497c340a0a391d00b7a53c1e3/node/src/chain_spec/turing.rs#L17-L20

Reply:
Ok, great idea! I will change it.